Rubocop didn't flag this, so I don't know how to catch it locally.
```
lib/revision_score_api_handler.rb:65: trailing whitespace.
+    completed_score['features'] = 
Error: Process completed with exit code 2.
```

--- a/lib/revision_score_api_handler.rb
+++ b/lib/revision_score_api_handler.rb
@@ -54,25 +54,16 @@ class RevisionScoreApiHandler
   def complete_score(score)
     completed_score = {}
 
-    # If the score already has error is key is because some API request failed some way
+    # If the score already has error key is because some API request failed some way
     completed_score['error'] = score.key?('error')
 
-    # Fetch the value for 'wp10' and 'prediction', or default to nil if not present.
-    completed_score['wp10'] = score.fetch('wp10', nil)
-    completed_score['prediction'] = score.fetch('prediction', nil)
     # Fetch the value for 'deleted, or default to 'false if not present.
     completed_score['deleted'] = score.fetch('deleted', false)
 
-    # Ensure 'features' has the correct value (a hash).
-    # For Wikidata, 'features' has to contain the LiftWing features.
+    # features field has to contain the reference-counter scores if
+    # different from nil. Otherwise, it should be the empty hash.
     completed_score['features'] =
-      if @wiki.project == 'wikidata'
-        score.fetch('features', {})
-      else
-        # For other wikis, 'features' has to contain the reference-counter scores if
-        # different from nil. Otherwise, it should be the empty hash.
-        score.fetch('num_ref').nil? ? {} : { 'num_ref' => score['num_ref'] }
-      end
+      score.fetch('num_ref').nil? ? {} : { 'num_ref' => score['num_ref'] }
 
     completed_score
   end

--- a/spec/lib/revision_score_api_handler_spec.rb
+++ b/spec/lib/revision_score_api_handler_spec.rb
@@ -18,10 +18,10 @@ describe RevisionScoreApiHandler do
 
       it 'returns completed scores if data is retrieved without errors' do
         expect(subject).to be_a(Hash)
-        expect(subject.dig('829840090')).to eq({ 'wp10' => nil, 'error' => false,
-        'features' => { 'num_ref' => 132 }, 'deleted' => false, 'prediction' => nil })
-        expect(subject.dig('829840091')).to eq({ 'wp10' => nil, 'error' => false,
-        'features' => { 'num_ref' => 1 }, 'deleted' => false, 'prediction' => nil })
+        expect(subject.dig('829840090')).to eq({ 'error' => false,
+        'features' => { 'num_ref' => 132 }, 'deleted' => false })
+        expect(subject.dig('829840091')).to eq({ 'error' => false,
+        'features' => { 'num_ref' => 1 }, 'deleted' => false })
       end
 
       it 'returns completed scores if there is an error hitting ReferenceCounterApi' do
@@ -29,10 +29,10 @@ describe RevisionScoreApiHandler do
           .to_raise(Errno::ETIMEDOUT)
 
         expect(subject).to be_a(Hash)
-        expect(subject.dig('829840090')).to eq({ 'wp10' => nil, 'error' => true,
-        'features' => {}, 'deleted' => false, 'prediction' => nil })
-        expect(subject.dig('829840091')).to eq({ 'wp10' => nil, 'error' => true,
-        'features' => {}, 'deleted' => false, 'prediction' => nil })
+        expect(subject.dig('829840090')).to eq({ 'error' => true,
+        'features' => {}, 'deleted' => false })
+        expect(subject.dig('829840091')).to eq({ 'error' => true,
+        'features' => {}, 'deleted' => false })
       end
     end
   end
@@ -50,9 +50,8 @@ describe RevisionScoreApiHandler do
 
     it 'returns deleted: true and empty features' do
       result = handler.get_revision_data [829840092]
-      expect(result.dig('829840092')).to eq({ 'wp10' => nil, 'error' => false,
-                                              'features' => {}, 'deleted' => true,
-                                              'prediction' => nil })
+      expect(result.dig('829840092')).to eq({ 'error' => false,
+                                              'features' => {}, 'deleted' => true })
     end
   end
 
@@ -87,20 +86,20 @@ describe RevisionScoreApiHandler do
     describe '#get_revision_data' do
       it 'returns completed scores if retrieves data without errors' do
         expect(subject).to be_a(Hash)
-        expect(subject.dig('157412237')).to eq({ 'wp10' => nil, 'error' => false,
-        'features' => { 'num_ref' => 111 }, 'deleted' => false, 'prediction' => nil })
-        expect(subject.dig('157417768')).to eq({ 'wp10' => nil, 'error' => false,
-        'features' => { 'num_ref' => 42 }, 'deleted' => false, 'prediction' => nil })
+        expect(subject.dig('157412237')).to eq({ 'error' => false,
+        'features' => { 'num_ref' => 111 }, 'deleted' => false })
+        expect(subject.dig('157417768')).to eq({ 'error' => false,
+        'features' => { 'num_ref' => 42 }, 'deleted' => false })
       end
 
       it 'returns completed scores if there is an error hitting reference-counter api' do
         stub_request(:any, /.*reference-counter.toolforge.org*/)
           .to_raise(Errno::ETIMEDOUT)
         expect(subject).to be_a(Hash)
-        expect(subject.dig('157412237')).to eq({ 'wp10' => nil, 'error' => true,
-        'features' => {}, 'deleted' => false, 'prediction' => nil })
-        expect(subject.dig('157417768')).to eq({ 'wp10' => nil, 'error' => true,
-        'features' => {}, 'deleted' => false, 'prediction' => nil })
+        expect(subject.dig('157412237')).to eq({ 'error' => true,
+        'features' => {}, 'deleted' => false })
+        expect(subject.dig('157417768')).to eq({ 'error' => true,
+        'features' => {}, 'deleted' => false })
       end
     end
   end

